rudra/deployments/emr_scripts/emr_config.py

```diff
@@ -62,6 +63,12 @@ def get_config(self):
 
         execute_training_code = ['python3.6'.format(self.repo_dir),
                                  training_file, self.hyper_params]
+
+        if self.ecosystem=='npm':
```

Comment:
It's alway better to use `if-else` loop. Please do the same for `execute_training_code`

Reply:
Ok, doing it :)